experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -1,65 +1,84 @@
         self.channel: str = channel
-        self.application_data = self.get_application_data(application, file_location)
-
-    def get_fml_clients(self, versions: list[str]) -> list[FmlClient]:
-        if not self.application_data:
-            return None
-        refs = self.get_version_refs(versions)
-        file_path = self.get_file_path()
-        return [self.create_client(file_path, self.channel, r) for r in refs]
+        self.application_data = self._get_application_data(application, file_location)
 
     @staticmethod
-    def get_application_data(application_name, file_location=BASE_PATH):
-        if os.path.exists(file_location):
+    def _get_application_data(application_name, file_location=BASE_PATH):
+        """
+        Fetch app data from local apps.yaml file to find application
+        names, github repos, fml paths within the repos, and release
+        versions.
+        """
+        if Path.exists(file_location):
             with open(file_location) as application_yaml_file:
                 file = yaml.safe_load(application_yaml_file.read())
                 if application_name in file:
                     return file[application_name]
                 else:
                     return None
 
-    def get_file_path(self):
+    def _get_local_file_path(self):
+        """
+        Get path to release feature manifest from experimenter (local).
+        """
+        path = Path(self.MANIFEST_PATH) / f"{self.application}" / "release.fml.yaml"
+        if Path.exists(path):
+            return path
+
+    def _get_remote_file_path(self):
+        """
+        Get path to the feature manifest from location defined in apps.yaml.
+        """
         if not self.application_data:
             return None
-        return Path(self.application_data["repo"]) / self.application_data["fml_path"]
+        return (
+            Path(self.application_data["repo"]["name"])
+            / self.application_data["fml_path"]
+        )
 
-    def get_version_refs(self, versions):
+    # Todo: Add versioning https://mozilla-hub.atlassian.net/browse/EXP-3875
+    def _get_version_refs(self, versions):
+        """
+        Get github refs for each version.
+        """
         if versions == []:
             return ["main"]
         refs = []
         for v in versions:
             version = version_packaging.parse(v)
             # Todo: this can be expanded later to fetch both the
             # branch/major version and the tagged minor version.
-            version_ref_minor = self.get_minor_version_ref(
+            version_ref_minor = self._get_minor_version_ref(
                 version,
             )
             refs.append(version_ref_minor)
         return refs
 
-    def get_major_version_ref(self, version):
+    def _get_major_version_ref(self, version):
         if not self.application_data:
             return None
         return self.application_data["major_release_branch"].format(
```

Comment:
We talked in slack and you said you weren't using the refs because we aren't communicating with GitHub.

These should be removed because these fields from apps.yaml are being removed in an upcoming PR.

Reply:
I was going to explicitly remove them in a different PR (maybe it makes sense to remove them in the same PR that the apps.yaml is removed?)

Reply:
#9654

Reply:
Yeah I can remove these in my pr.